WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -176,8 +176,15 @@ + (NSArray *)routes
   NSString *elementUUID = request.parameters[@"uuid"];
   XCUIElement *element = [elementCache elementForUUID:elementUUID];
   NSError *error = nil;
-  if (![element fb_tapWithError:&error]) {
-    return FBResponseWithError(error);
+  //if element is not hittable, fb_tapWithError returns with success=YES
+  if (element.isHittable) {
```

Comment:
Do you think it would make sense to move it into `fb_tapWithError`? Then we could easily test it.